packages/run-protocol/src/vaultFactory/vault.js

```diff
@@ -285,26 +316,27 @@ export const makeInnerVault = (
     // [https://github.com/Agoric/dapp-token-economy/issues/123]
     const collateralizationRatio = await getCollateralizationRatio();
     /** @type {VaultUIState} */
-    const uiState = snapshotState(vaultState, collateralizationRatio);
+    const uiState = snapshotState(vaultPhase, collateralizationRatio);
     trace('updateUiState', uiState);
 
-    switch (vaultState) {
-      case VaultState.ACTIVE:
-      case VaultState.LIQUIDATING:
+    switch (vaultPhase) {
+      case VaultPhase.ACTIVE:
+      case VaultPhase.LIQUIDATING:
```

Comment:
add `TRANSFERABLE` here.

If no tests broke, then there aren't any tests for transfering. Please add some tests in that case.

Reply:
There are tests for transferring. "Transfer" is only visible in `finish` event that is generated when the transfer is started

Reply:
This is a smell though that the transfer case is so special. Refactored in 29b9973948516b025bbbb905a2b8391cb5629af3

Reply:
I hit some snags so intend to punt https://github.com/Agoric/agoric-sdk/commit/fcdd7a03747c8b5fedf7b97994e7a9c55989cdab until https://github.com/Agoric/agoric-sdk/issues/4415